libs/src/services/web3Manager/index.js

```diff
@@ -154,7 +154,8 @@ class Web3Manager {
           contractRegistryKey,
           contractAddress,
           this.ownerWallet.getAddressString(),
-          encodedABI
+          encodedABI,
+          gasLimit !== DEFAULT_GAS_AMOUNT ? gasLimit : null
```

Comment:
why not just always pass the gasLimit here?

Reply:
this is a good point and it was bc the DEFAULT_GAS_AMOUNT didn't match identity, but the right thing to do is to update those to be equal

Reply:
word. also the gaslimit vs gasamount vs gas terminology is pretty confusing but i think web3's terminology around this is the source of the confusion. feels like this should really be just gas or gasamount and gaslimit should be reserved for blocks only? but doesn't need to be fixed right now

Reply:
its the txGasLimit, not gasAmount though - it is just the max not the expected amount

Reply:
yeah it is, but web3 describes it as an amount not a limit. that's why i said i think the source of confusion is from web3
[from their docs](https://web3js.readthedocs.io/en/v1.2.1/web3-eth.html#sendtransaction)
> gas - Number: (optional, default: To-Be-Determined) The amount of gas to use for the transaction (unused gas is refunded).